apps/wolfsshd/wolfsshd.c

```diff
@@ -2396,13 +2396,13 @@ static int StartSSHD(int argc, char** argv)
                 conn->fd = (int)accept(listenFd, (struct sockaddr*)&clientAddr,
                     &clientAddrSz);
                 if (conn->fd >= 0) {
-                    inet_ntop(AF_INET,
 #ifdef TEST_IPV6
-                              &clientAddr.sin6_addr,
+                    inet_ntop(AF_INET, &clientAddr.sin6_addr, conn->ip,
```

Comment:
This doesn't actually work. inet_ntop() is going to take the first 32-bits of sin6_addr and turn it into a dotted-quad string.